.changeset/early-dancers-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fiori-freestyle-writer': patch
```

Comment:
Should be minor at least since the behaviour is changed and a feature removed.

Reply:
Done in 0f829edc6fa15e3886af6ccfd3642f25e16ec461 . Thanks.